corehq/apps/accounting/utils.py

```diff
@@ -116,10 +116,9 @@ def domain_has_privilege(domain, privilege_slug, **assignment):
     from corehq.apps.accounting.models import Subscription
     try:
         plan_version = Subscription.get_subscribed_plan_by_domain(domain)[0]
```

Comment:
Since there will be lot of different permission-checks, could `Subscription.get_subscribed_plan_by_domain(domain)[0]` be cached at the request level and passed here, instead of fetching each time? (Does this sound correct)